platform/broadcom/docker-syncd-brcm/syncd.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+function clean_up {
+    service syncd stop
+    exit
+}
+
+trap clean_up SIGTERM SIGKILL
+
+service syncd start
```

Comment:
syncd has a complex init script. I don't have much experience with syncd operations and testing methods, so I figured this task would be best left to someone with more experience. Init script functionality just needs to be ported into the syncd.sh script I created.